Chethz/Pacman/PacmanTests/GridTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+using PacmanLibrary;
+
+namespace PacmanTests
+{
+    class GridTests
+    {
+            private Grid _grid;
+
+            [SetUp]
+            public void Setup()
+            {
+                _grid = new Grid(5, 5);
+            }
+
+            [Test]
+            [TestCase(0, 0, true)]
+            [TestCase(4, 4, true)]
+            [TestCase(5, 5, false)]
+            [TestCase(-1, -1, false)]
```

Comment:
You should really separate the test cases for testing out of bounds for x and y separately e.g.
```c#
[TestCase(5, 4, false)]
[TestCase(4, 5, false)]
[TestCase(-1, 0, false)]
[TestCase(0, -1, false)]
```